### PR TITLE
Delegate opa logger to logp lib

### PR DESCRIPTION
**Pull request by amirbenun**
Resolves https://github.com/elastic/security-team/issues/4168

### CODE REVIEW

File: evaluator/logger.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+func (l *logger) GetLevel() logging.Level {
+	return toOpaLevel(l.lvl.Level())
+}
+
+func (l *logger) SetLevel(level logging.Level) {
+	l.lvl.SetLevel(toZapLevel(level))
+}
+
+func mapToArray(m map[string]interface{}) []interface{} {
+	ret := []interface{}{}
```
Comment (oren-zohar) [inline evaluator/logger.go:60]:
```suggestion
	var ret []interface{}
```
as a rule of thumb, we should try to use nil slice declaration wherever we can so an empty result will be comparable with nill, see an [exmaple](https://go.dev/play/p/V8pesYrwZBY)

File: evaluator/logger.go [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+	return zap.FatalLevel
+}
+
+func toOpaLevel(l zapcore.Level) logging.Level {
+	if res, ok := zapToOpaLevelsMap[l]; ok {
+		return res
+	}
+
+	return logging.Level(4)
```
Comment (oren-zohar) [inline evaluator/logger.go:94]: what does 4 stand for? level.Debug is 3

**Review by oren-zohar — CHANGES_REQUESTED** [review]
Looks good, need to generate license headers with `mage addLicenseHeaders`. Did you test it? can you please share how the logs look now?

**Comment by amirbenun** [comment]
Fleet UI logs screenshot:
image

Note that "Bundle load failed..." is produced by OPA.

**Review by oren-zohar — APPROVED** [review]
🚀 🚀 🚀 🚀

**Pull request merged**